src/migrations/migrate_sync_visible_flag.js

```diff
@@ -0,0 +1,16 @@
+import { isLocalStorageAvailable } from '../util/settings_persister';
+
+// Fixes a typo in the config: `visibile` !== `visible`
+export default () => {
+  if (!isLocalStorageAvailable) {
+    return;
+  }
+
+  const shouldSyncOnBecomingVisible = localStorage.getItem('shouldSyncOnBecomingVisibile');
+  if (!shouldSyncOnBecomingVisible) {
+    return;
```

Comment:
So if the old value is `false` we don't want to copy that to the new value?

Reply:
Hm.. atm it doesn't make a difference, because the default isn't set, so it's falsy. But that might change in the future and people who explicitly turned it off will lose that setting.

Good call. I'm removing the guard.